web/themes/custom/vote_gov/templates/node/node--state-territory--full.html.twig

```diff
@@ -8,46 +8,17 @@
 {% extends 'node.html.twig' %}
```

Comment:
We need to add the `voter-info` class to the wrapper to make the top margin to display correctly.